data/_models/shared/Catalogues.csv

```diff
@@ -1,5 +1,8 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
 Catalogues,,,,,,,,,,,dcat:Catalog,A collection of resources within a network or consortium or about a common topic,DataCatalogueFlat,
+Catalogues,,name,,1,true,,,,,,"dterms:title,dct:title",The name given to the Catalogue,DataCatalogueFlat,
```

Comment:
We are using dct and dcterms (and dterms?) intermittently throughout the annotations. Can we use 'dct:', as is done by DCAT-AP: https://semiceu.github.io/DCAT-AP/releases/3.0.0/#used-prefixes